python/Produce_Config_File.py

```diff
@@ -115,6 +114,23 @@ def CyclicColor(DrawArg):
 		pass
 
 	text_file.write("\t\t\t[END_DATA]\n" )
+
+	if args.Fit=='true':
+		text_file.write("\t\t\t[BEGIN_FIT]\n" )
+		text_file.write("\t\t\tFit_Color = '%s';\n"%(CyclicColor(DrawArg)))
+		text_file.write("\t\t\tFit_Formula = '%s';\n"%(args.FitFormula))
+		text_file.write("\t\t\tFit_LegEntry = 'Fit_%s';\n"%(legEntry))
+		text_file.write("\t\t\tFit_Line_Size = '%s';\n"%(args.FitLineSize))
+		text_file.write("\t\t\tFit_Line_Style = '%s';\n"%(args.FitLineStyle))
+		text_file.write("\t\t\tFit_Name = 'Fit_%s';\n"%(legEntry))
+		text_file.write("\t\t\tFit_Option = '%s';\n"%(args.FitOption))
+		text_file.write("\t\t\tFit_Param_IGuess = '%s';\n"%(args.FitParamIGuess))
+		text_file.write("\t\t\tFit_Perform = '%s';\n"%(args.FitPerform))
+		text_file.write("\t\t\tFit_Range = '%s';\n"%(args.FitRange))
+		text_file.write("\t\t\t[END_FIT]\n" )
+		pass
+
+	DrawArg=DrawArg+1
```

Comment:
Get into the habit of using operators to do this, the code is cleaner when they are used:

```
>>> iValue=1
>>> print iValue
1
>>> iValue+=1
>>> print iValue
2
```

For a list of basic operators see:
https://www.tutorialspoint.com/python/python_basic_operators.htm